pkg/server/datastore/sqlstore/sqlstore_test.go

```diff
@@ -3904,6 +3932,8 @@ func (s *PluginSuite) TestMigration() {
 				prepareDB(true)
 				require.True(s.ds.db.Dialect().HasColumn("attested_node_entries", "can_reattest"))
 				require.True(s.ds.db.Dialect().HasColumn("registered_entries", "hint"))
+			case 18:
```

Comment:
I think this might be merge cruft? We don't need a migration test from version 18, since that is the latest. The addition of the can_reattest column is being tested in the migration test from version 17 (just above here).

Reply:
You're right, this is not needed